components/atom/tag/src/index.js

```diff
@@ -9,15 +9,20 @@ const ACTIONABLE_ONLY_PROPS = [
   'iconPlacement',
   'target',
   'actionable',
-  'linkFactory'
+  'linkFactory',
+  'rel'
 ]
 const STANDARD_ONLY_PROPS = ['closeIcon', 'onClose']
 const SIZES = {
   LARGE: 'large',
   MEDIUM: 'medium',
   SMALL: 'small'
 }
-
+const LINK_TYPES = {
+  NO_FOLLOW: 'nofollow',
+  NO_OPENER: 'noopener',
+  NO_REFERRER: 'noreferrer'
```

Comment:
There are more common link types, for example `prev` and  `next`, see: https://www.w3schools.com/TAGS/att_a_rel.asp

Reply:
Sure, we can add more link types, do you miss some others more than prev and next?